package.json

```diff
@@ -16,8 +16,8 @@
     "pre-commit": "lint-staged --concurrent false",
     "pre-push": "pnpm build",
     "prepare": "husky install && chmod +x .husky/*",
-    "prettier": "prettier . --loglevel warn --write",
-    "prettier:check": "prettier . --loglevel warn --check",
```

Comment:
I previously added `--loglevel warn` because it lists _all_ the files otherwise, even the ones that are correctly formatted. Do you like that behavior, or does adding `--cache` and `--list-different` change it?

Reply:
it doesn't list all files, it updates only last line with linted file
<img width="504" alt="image" src="https://user-images.githubusercontent.com/7361780/234666480-5ce5fd95-6be7-46c2-8a29-a72dceb9f179.png">